content/post/2020-07-24-data-science-tutorials-with-learnr-and-gradethis.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Data Science Tutorials with learnr and gradethis
+author: Lee Suddaby, Zeno Kujawa
+date: '2020-07-24'
+slug: data-science-tutorials-with-learnr-and-gradethis
+categories: 
+  - teaching
+  - shiny
+  - rstats
+tags: 
+  - learnr
+  - gradethis
+  - tidyverse
+keywords:
+  - tech
+---
+
+*This post was contributed by [Lee Suddaby](https://github.com/lee-suddaby) and [Zeno Kujawa](https://github.com/ZenoMK), second year students at the University of Edinburgh majoring in Mathematics and Data Science, respectively.*
```

Comment:
@ZenoMK If you want to change the introduction for yourself  (or any other part of the post of course) just do so. Feel free to change the link to your profile if there's something else that's more appropriate - I wasn't sure when I wrote it.